README.md

```diff
@@ -1,2 +1,443 @@
-# graviton
-verify your TEAL program by experiment and observation
+<!-- markdownlint-disable no-inline-html -->
+<!-- markdownlint-disable ol-prefix -->
+
+# GRAVITON (aka the TEAL Blackbox Toolkit): Program Reporting and Testing via Dry Runs
```

Comment:
@tzaffi @jasonpaulos What support guarantees does graviton provide?

* My default preference is to add a _brief_ README disclaimer like:  Graviton is subject to change and makes no backwards compatibly guarantees.
* I think a broader conversation exists for support guarantees beyond graviton.  I'm trying to avoid the broader conversation while ensuring we retain flexibly in graviton.  I'm flexible on discussion specific to graviton.

Reply:
👍